markymark/Classes/Renderer/Configurations/MarkdownToViewConverterConfiguration.swift

```diff
@@ -10,24 +10,42 @@ import UIKit
 
 open class MarkdownToViewConverterConfiguration: MarkDownConverterConfiguration<UIView> {
 
+    public var urlOpener: URLOpener? {
+        didSet {
+            markDownItemToLayoutBuilderMap.values.forEach(setUrlOpenerIfNeeded)
+        }
+    }
+
     public init(elementComposer: ElementComposer<UIView>, styling: Styling, urlOpener: URLOpener? = nil) {
 
```

Comment:
You should still set the local urlOpener to the passed urlOpener. Right now urlOpener is unused causing the initial urlOpener never to be set.